python/helpers.py

```diff
@@ -157,30 +156,36 @@ def extract_tokens(token_id, rpc_url, json_id=None, tree_id=None):
             "showInscription": False,
             },
         },
-        }
+        }, timeout = 10
```

Comment:
we prob dont need this right

Reply:
if we don't have this the rpc could technically make our whole thread hang forever if it doesn't respond